### ISSUE TITLE
assigning zero weights to hourglass decoder while predicting mask

### ISSUE THREAD

**Issue by saandeepa93** [AliaksandrSiarohin/mtm#27]
Hello,

In dense motion module, while running the predictions through the hourglass to calculate the mask, the hourglass' decoder network's weights are assigned to 0 and then the bias is initialized to a particular value. 

_Can you please let me know the reason for this as I am trying to correlate this with the original paper?_
**modules -> dense_motion_module.py** 
self.hourglass.decoder.conv.weight.data.zero_()
self.hourglass.decoder.conv.bias.data.copy_(torch.tensor(bias_init, dtype=torch.float))

**Comment by AliaksandrSiarohin** [comment]
This is just a way to have everything assigned to the background (so no motion initially).
